@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -323,18 +324,15 @@ export class TileGeometryCreator {
 
         this.createLabelRejectionElements(tile, decodedTile);
 
-        // HARP-7899, disable ground plane for globe
-        if (tile.dataSource.addGroundPlane && tile.projection.type === ProjectionType.Planar) {
-            // The ground plane is required for when we change the zoom back and we fall back to the
-            // parent, in that case we reduce the renderOrder of the parent tile and this ground
-            // place ensures that parent doesn't come through. This value must be above the
-            // renderOrder of all objects in the fallback tile, otherwise there won't be a proper
-            // covering of the parent tile by the children, hence dividing by 2. To put a bit more
-            // concretely, we assume all objects are rendered with a renderOrder between 0 and
-            // FALLBACK_RENDER_ORDER_OFFSET / 2, i.e. 10000. The ground plane is put at -10000, and
-            // the fallback tiles have their renderOrder set between -20000 and -10000
-            TileGeometryCreator.instance.addGroundPlane(tile, -FALLBACK_RENDER_ORDER_OFFSET / 2);
-        }
+        // The ground plane is required for when we change the zoom back and we fall back to the
+        // parent, in that case we reduce the renderOrder of the parent tile and this ground
+        // place ensures that parent doesn't come through. This value must be above the
+        // renderOrder of all objects in the fallback tile, otherwise there won't be a proper
+        // covering of the parent tile by the children, hence dividing by 2. To put a bit more
+        // concretely, we assume all objects are rendered with a renderOrder between 0 and
+        // FALLBACK_RENDER_ORDER_OFFSET / 2, i.e. 10000. The ground plane is put at -10000, and
+        // the fallback tiles have their renderOrder set between -20000 and -10000
+        this.addGroundPlane(tile, TileGeometryCreator.instance.fallBackRenderOffset, true);
```

Comment:
Will this work for globe? I guess we would need to subdivide the ground plane the same way we subdivide the actual geometry.

Reply:
Never mind, just saw that you moved the check inside the function

Reply:
I moved the check inside the `addGroundPlane` method, because otherwise I would need the check duplicated.

Reply:
Because I need to call the `addGroundPlane` twice, once when it is empty, and then once in the normal condition.